Jamfile

```diff
@@ -766,6 +766,7 @@ lib torrent
 
 	: # default build
 	<threading>multi
+	<c++-template-depth>512
```

Comment:
Other versions only pass because they just barely manage to stay below the depth limit. In the future something else may trip the limit everywhere so we might as well raise it now.